src/v2/view-builder/views/email/BaseAuthenticatorEmailView.js

```diff
@@ -77,20 +77,30 @@ const Body = BaseFormWithPolling.extend(Object.assign(
       BaseForm.prototype.triggerAfterError.apply(this, arguments);
       this.stopPolling();
 
-      // Wait 1 min before polling again & do not show error message when encounter rate limit error
-      if (error.responseJSON?.errorSummaryKeys?.includes('tooManyRequests') ||
-        error.responseJSON?.errorCode === 'E0000047') {
+      if (error.responseJSON?.errorSummaryKeys?.includes('idx.session.expired')) {
+        // Do NOT resume polling since session is invalid and polling is already stopped
+        return;
+      }
+
+      if (error.responseJSON?.errorSummaryKeys?.includes('tooManyRequests')
```

Comment:
To avoid unnecessary empty `return`  we can simplify the code a little bit.
```
if( isRateLimitErrors()) {
       setTimeout(() => {
         model.trigger('clearFormError');
        }, 0);
        setTimeout(() => {
          this.startPolling(this.options.appState.get('dynamicRefreshInterval'));
        }, 60000);
} else {
 this.startPolling(this.options.appState.get('dynamicRefreshInterval'));
}


isRateLimitErrors() {
return (error.responseJSON?.errorSummaryKeys?.includes('tooManyRequests') ||
    error.responseJSON?.errorCode === 'E0000047') &&
    !error.responseJSON?.errorIntent
}
```

Reply:
I was trying to avoid long chained conditions but sure